utils/utils.go

```diff
@@ -35,11 +38,86 @@ var log = logf.Log.WithName("openliberty_utils")
 const serviceabilityMountPath = "/serviceability"
 const ssoEnvVarPrefix = "SEC_SSO_"
 const OperandVersion = "1.3.2"
-const ltpaKeysMountPath = "/config/managedLTPA"
-const ltpaServerXMLOverridesMountPath = "/config/configDropins/overrides/"
+
+// LTPA constants
+const managedLTPAMountPath = "/config/managedLTPA"
 const LTPAServerXMLSuffix = "-managed-ltpa-server-xml"
-const ltpaKeysFileName = "ltpa.keys"
-const ltpaXMLFileName = "managedLTPA.xml"
+const LTPAServerXMLMountSuffix = "-managed-ltpa-mount-server-xml"
+const LTPAKeysFileName = "ltpa.keys"
+const LTPAKeysXMLFileName = "managedLTPA.xml"
+const LTPAKeysMountXMLFileName = "managedLTPAMount.xml"
+const LTPAKeysCreationScriptFileName = "create_ltpa_keys.sh"
+
+// Mount constants
+const SecureMountPath = "/output/resources/liberty-operator"
+const overridesMountPath = "/config/configDropins/overrides"
+
+// Password encryption constants
+const ManagedEncryptionServerXML = "-managed-encryption-server-xml"
+const ManagedEncryptionMountServerXML = "-managed-encryption-mount-server-xml"
+const PasswordEncryptionKeySuffix = "-wlp-password-encryption-key"
```

Comment:
Let's use `wlp-password-encryption-key` as the user-provided secret name, so that we go out the door with parity between WLO and OLO for this new feature. 

In the future, we can consider allowing the secret with operator prefix higher precedence (i.e. olo-wlp-password-encryption-key), but don't think that's necessary now.